.github/workflows/ci.yml

```diff
@@ -15,6 +19,22 @@ jobs:
         architecture: 'x64'
     - name: Build
       run: |
+        echo "Downloading Spark"
+        wget -q https://archive.apache.org/dist/spark/spark-2.1.1/spark-2.1.1-bin-hadoop2.7.tgz
+        tar xfz spark-2.1.1-bin-hadoop2.7.tgz
+        rm spark-2.1.1-bin-hadoop2.7.tgz
+        export SPARK_HOME=`pwd`"/spark-2.1.1-bin-hadoop2.7/"
```

Comment:
Setting up Spark this way seems to be the preferred way to ensure that pyspark and spark versions match.